examples/eigenproblems/eigenproblems_ex3/eigenproblems_ex3.C

```diff
@@ -110,9 +110,9 @@ int main (int argc, char ** argv)
   libmesh_example_requires(false, "--disable-singleprecision");
 #endif
 
-#ifdef LIBMESH_USE_COMPLEX_NUMBERS
-  // SLEPc currently gives us an "inner product not well defined" with
-  // Number==complex
+#ifdef LIBMESH_USE_COMPLEX_NUMBERS && SLEPC_VERSION_LESS_THAN(3,6;2)
+  // SLEPc may give an "inner product not well defined" with
+  // Number==complex; but for me it worked fine with vs 3.6.2
   libmesh_example_requires(false, "--disable-complex");
```

Comment:
You may want to update the help string here to "--disable-complex and SLEPc >= 3.6.2"

Reply:
sure, this is of course better.